student.rb

```diff
@@ -8,21 +13,6 @@ def initialize(options = {})
     @name = options.fetch(:name, "")
   end
```

Comment:
A lot of the initialization and `attr`s are common to both teachers and students.  How could you refactor those to DRY things up?